### PR TITLE
Feat/dynamic supplier dashboard

### PR DESCRIPTION
**Pull request by PaMarzec**
## Description
* changed the dashboard to be dynamic 
* added mock data for "customer", "material" and "your stock"
* added the changing of backgroundcolor to red if production is lower than total demand

### CODE REVIEW

File: frontend/src/views/SupplierDashboard.vue [hunk 1]
```diff
@@ -28,27 +28,29 @@
...
                     </div>
-                    <select onload="addOptionToDropdown(dropdown-material,dropdownMaterialData);" name="ddm" id="dropdown-material" class="w-60 py-2 px-4 bg-gray-200 text-gray-700 border border-gray-200 rounded focus:bg-white focus:outline-none focus:border-gray-500">
-                        <option value="central-control-unit">Central Control Unit</option>
-                        <option value="steering-wheel">Steering Wheel</option>
-                        <option value="wheel">Wheel</option>
+                    <select v-model="dropdownMaterial" id="dropdown-material" name="ddm"  class="w-60 py-2 px-4 bg-gray-200 text-gray-700 border border-gray-200 rounded focus:bg-white focus:outline-none focus:border-gray-500">
+                        <option disabled value="" selected hidden>Choose a material</option>
+                        <option v-for="item in dropdownCustomer.materials " :value="item" @click="emptyTotalDemandArray()">{{item.name}}</option>
                     </select>
+                    <!--
```
Comment (tom-rm-meyer-ISST) [inline frontend/src/views/SupplierDashboard.vue:44]: please remove unused code

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Please update small fix and then let me know. For the next time: when doing a PR, please don't remove the second part of the PR. It reminds you of then performing the dashtool check and checking for missing license headers. If you did not check that, please also do :)

**Review by tom-rm-meyer-ISST — APPROVED** [review]
Thanks for fixing! @mhellmeier please merge.

**Review by tom-rm-meyer-ISST — APPROVED** [review]

**Review by mhellmeier — APPROVED** [review]
LGTM, since the failed Q-Gate pipeline is already addressed.

**Pull request merged**